xbmc/video/VideoLibraryQueue.cpp

```diff
@@ -32,7 +32,7 @@
 using namespace std;
 
 CVideoLibraryQueue::CVideoLibraryQueue()
-  : CJobQueue(false, 1, CJob::PRIORITY_LOW_PAUSABLE),
```

Comment:
Please read the PR discussion. I simply restored the same behaviour as before #6406. We can still discuss if we want to change this.